tests/v2_validation/cattlevalidationtest/core/test_ldap_auth.py

```diff
@@ -1590,3 +1592,126 @@ def test_secret_setting(admin_client):
                               password=ldap_main_pass)
     secret = client.by_id_setting('api.auth.ldap.service.account.password')
     assert secret.value is None
+
+
+# 2
+@if_test_ad
+def test_ad_restrict_to_specific_user(admin_client):
+    ldap_main_user = os.environ.get('AD_MAIN_USER')
+    ldap_main_pass = os.environ.get('AD_MAIN_PASS')
+    ldap_user2 = os.environ.get('AD_USER2')
+    ldap_pass2 = os.environ.get('AD_PASS2')
+    ldap_user3 = os.environ.get('AD_USER3')
+    ldap_pass3 = os.environ.get('AD_PASS3')
+    ldap_port = os.environ.get('LDAP_PORT')
+
+    user = ADMIN_TOKEN['userIdentity']
+    allowed_identities = []
+    allowed_identities.append(user)
+    token2 = get_authed_token(username=ldap_user2,
```

Comment:
I see, so i should call `client.list_identity()` to list these users identities?